ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallFooter.kt

```diff
@@ -62,33 +62,3 @@ fun PaywallFooter(
         }
     }
 }
-
-@Suppress("MagicNumber")
-@Preview(showBackground = true)
-@Composable
-private fun PaywallFooterPreview() {
```

Comment:
These seem to have been broken for a while... Basically we're not mocking the view model, so it can't work on the previews. We do have footer previews for all templates (and the default paywall now) so I think we should be mostly covered...